dashboard/ui.R

```diff
@@ -82,25 +82,44 @@ navbarPage(
       
       # Show a plot of the generated distribution
       mainPanel(
-        fluidRow(column(width = 8,
-                        box(
-                          plotOutput("top20bar", height = 800),
-                          width = NULL),
-                        ),
-                 column(width = 4,
-                        box(
-                          plotOutput("n_indicator", height = 200), 
-                          width = NULL
-                        ),
-                        box(
-                          plotlyOutput("gender_donut", height = 300),
-                          width = NULL),
-                        box(plotlyOutput("race_donut", height = 225),
-                            width = NULL))
-                   
-                 )))
+        tabsetPanel(type = "tabs",
+                    tabPanel("Overall",
+                             fluidRow(column(width = 8,
+                                             box(
+                                               plotOutput("top20bar", height = 800),
+                                               width = NULL),
+                             ),
+                             column(width = 4,
+                                    box(
+                                      plotOutput("n_indicator", height = 200), 
+                                      width = NULL
+                                    ),
```

Comment:
I see that this box is technically smaller than the donut charts, but the arc still appears bigger.  Might make this even smaller, or add some padding between it and the donut.